task.js

```diff
@@ -0,0 +1,30 @@
+function reverseListDrinks(){
+    let usersListElement = document.querySelector('.drinks-list');
+  
+    let listElements = [];
+  
+    let node = usersListElement.firstElementChild ;
+    while(node != null) { 
+      listElements.push(node.textContent);
+      node = node.nextElementSibling;
+    }; 
+  
+    // sorted and filtered list drinks
+    let sortListElements =  Array.from(new Set(listElements)).sort(); 
+  
+    // delete elements in sector ul
+    usersListElement.innerHTML = '';
+  
+    sortListElements.map( drink => {
+        let li = document.createElement('li');
+        li.innerHTML = `<p>${drink}</p>`;
+        return li;
+    }).forEach(liElem => {
+            liElem.classList.add('onedrink');
+            usersListElement.append(liElem);
+        }); 
+    // push new list dirks
+    document.body.append(usersListElement);
```

Comment:
a lot of code, that is hard to read. I recommend you this simple implementation
```suggestion
   sortListElements.forEach((value) => {
    usersListElement.innerHTML += `<li><p>${value}</p></li>`;
  });
```